bin/n

```diff
@@ -793,15 +797,15 @@ install() {
   log installing "${g_mirror_folder_name}-v$version"
 
   local url="$(tarball_url "$version")"
-  is_ok "${url}" || abort "download preflight failed for '$version' (${url})"
+  is_ok "${url}" || abort "download preflight failed for $version $(display_masked_url "${url}")"
```

Comment:
Preserving original style now I realise the single quotes and brackets appear in the output.
```suggestion
  is_ok "${url}" || abort "download preflight failed for '$version' ($(display_masked_url "${url}"))"
```